tools/eventlist/pkg/output/output_test.go

```diff
@@ -1083,18 +1084,19 @@ func TestPrintJSON(t *testing.T) { //nolint:golint,paralleltest
 		showStatistic bool
 	}
 	formatType := "json"
+	level := ""
 	tests := []struct {
 		name    string
 		args    args
 		wantErr bool
 	}{
-		{"test", args{filename: &o1, eventFile: &s10}, false},
+		{"test1", args{filename: &o1, eventFile: &s10}, false},
```

Comment:
Do we have any binary test files with level field set? I can then add test to test filtering.